modules/ocf/templates/firefox/prefs.js.erb

```diff
@@ -0,0 +1,12 @@
+lockPref("browser.startup.homepage", "<%= @desktop_homepage %>");
+pref("browser.cache.disk.capacity", 0);
+pref("browser.download.useDownloadDir", false);
```

Comment:
don't we want this true?

> True (default): Without asking a location, download files to the directory specified in browser.download.folderList

I think the Downloads folder is pretty standard at this point, people are going to get confused when they accidentally end up in `/` or something.